app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,48 @@
  *= require foundation_and_overrides
  */
 
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
```

Comment:
This works, but it will make the page load slower. It might be a good idea to download a font-pack for what you want from [Font Squirrel](http://www.fontsquirrel.com/tools/webfont-generator) and then install them: http://stackoverflow.com/questions/21250303/official-way-of-adding-custom-fonts-to-rails-4. You don't have to do it for this PR, necessarily, but it's a refactor opportunity.